# data

## Summary of strain names, run names and accessions for genome data in [Almeida et al 2015](http://onlinelibrary.wiley.com/doi/10.1111/mec.13341/abstract)
**EBI-ENA Data Archive Scerevisiae DB.tsv**

## Data used in [Tilakaratna and Bensasson](http://www.g3journal.org/content/7/9/2919)

### TilakaratnaBensasson17

**CEN1.tree** to **CEN16.tree**
 
These are 16 neighbour joining F84 distance trees in Newick format: one for each of 16 centromere loci in Saccharomyces cerevisiae. The bootstrap values shown are from a total of 10,000 bootstraps, and trees are unrooted. The 80 S. cerevisiae strains shown in each tree were completely homozygous and are fully described in Table S1 of Tilakaratna and Bensasson. 

**CEN1forcbyc.gfa** to **CEN1forcbyc.gfa**

These are 16 alignments in gapped fasta format for centromeres and flanking DNA used to generate the trees.

## Data used in Bensasson et al 2018

### TilakaratnaBensasson17

**Bensasson_etalTableS2.tsv**

Table S1 referred to in the manuscript. This is a table in text format with tab separated values summarizing heterozygosity analyses for every strain. This includes exact counts of high quality heterozygous base calls (highQualityHetCount); the total length of high quality sequence (highQualityLength; bases with a phred-scaled quality score over 40); the proportion of high quality heterozygous sites; the length of regions that have undergone Loss of Heterozygosity (LOHlength) assessed in 100 kb windows;  heterozygosity analysis after excluding LOH regions, centromeres and annotated repeats (annotationLohFilteredHetCount, annotationLohFilteredLength, annotationLohFilteredHeterozygosity); heterozygosity analysis after excluding LOH regions, centromeres and annotated repeats, and regions with more than double the expected read depth (depthFilteredHetCount, depthFilteredLength, depthFilteredHeterozygosity); heterozygosity analysis at 948,860 nucleotide sites that are common to all strains (sitesIn950kbHetCount, sitesIn950kbLength, sitesIn950kbHeterozygosity).
