# README

Data tables, alignments, trees and scripts that accompany the Scopel et al 2020 manuscript:
>"Genetic variation in aneuploidy prevalence and tolerance across the
Saccharomyces cerevisiae phylogeny"
>Eduardo F.C. Scopel<sup>1</sup>, James Hose<sup>2</sup>, Douda Bensasson<sup>1</sup>, Audrey P. Gasch<sup>2</sup>
><sup>1</sup> Institute of Bioinformatics and Department of Plant Biology, University of Georgia, Athens, Georgia, USA
><sup>2</sup>  Laboratory of Genetics and Center for Genomic Science Innovation, University of Wisconsin-Madison, Madison, WI, 53704 USA

## Data tables

**peter_scopel_20201002.txt** - Tab delimited text file with information on 1011 strains derived from Peter et al 2018 Table S1, estimated levels of heterozygosity, and allele calls at the SSD1 locus (see Methods).

**453strains_scopel_20201002.txt** - Tab delimited text file on a filtered subset of 453 strains from peter_scopel_20201002.txt after haploids, mosaics, close relatives etc (see Methods).

## Phylogenetic trees

**Fig6A.tree** - Maximum likelihood phylogenetic tree for the reduced set of 621 strains used for modeling (see Methods). 

**FigS3B.tree** - Maximum likelihood phylogenetic tree for 453 strains (422 diploids and 31 polyploids) after removing closely related strains (genetic distance < 0.000007, see Methods).

## Alignment files

**621.ssd1.gene.fa** - Fasta format DNA sequence alignment for the SSD1 gene for the 621 strains used in modeling and phylogenetic analyses.

## Scripts

**scopel_glm_DB.R** - R script that shows the logistic regression (glm) and other stats analyses mentioned in the text.

**.py** - A python script that reads a pairwise distance matrix generated by the dnadist function of PHYLIP (version 3.697; Felsenstein 1989, Felsenstein 2005) and randomly chooses a single strain from each group of similar strains given a user-defined genetic distance threshold.

**RD_indiv_file.R** - An R script ..
